### DISCUSSION TITLE
Suggest me a beginner friendly repo for open source

### DISCUSSION THREAD

**Discussion by Maniik113** [community/community#72600] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want an open-source repo to contribute.

**Comment by Maniik113** [comment]
@Manishak798 kindly answer

**Comment by Manishak798** [comment]
Ya sure follow the link [dive-to-dev](https://github.com/Manishak798/Dive-to-Dev_Hacktoberfest23)

**Comment by Manishak798** [comment]
@Maniik113 check
